--- tools/allocation_tracking/track_allocations.py    2016-03-17 22:29:19.000000000 -0400
+++ tools/allocation_tracking/track_allocations.py    2016-03-17 22:29:16.000000000 -0400
@@ -3,7 +3,7 @@
 import numpy as np
 import gc
 import inspect
-from alloc_hook import NumpyAllocHook
+from numpy_allocation_tracking.alloc_hook import NumpyAllocHook

 class AllocationTracker(object):
     def __init__(self, threshold=0):